README.md

```diff
@@ -210,6 +210,18 @@ repo](https://mesonbuild.com/Quick-guide.html#installation-from-source) and call
 commands in the root of the tree builds and installs all schedulers under
 `~/bin`.
 
+#### Static linking against libbpf submodule (preferred)
+
+ ```
+$ cd $SCX
+$ git submodule init
+$ make -C libbpf/src
+$ meson setup build --prefix ~ -D libbpf_a=$SCX/libbpf/src/libbpf.a
```

Comment:
Should we make this the default behavior, and instead allow people to override `libbpf_a` to something else?

Reply:
That makes sense. Will update

Reply:
Agreed. Ideally the build script can minimize the pain of working with submodules by hiding the details when possible.